src/main/java/com/graphhopper/converter/api/GisgraphySearchEntry.java

```diff
@@ -0,0 +1,224 @@
+package com.graphhopper.converter.api;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.graphhopper.converter.data.CountryInfo;
+
+/**
+ * @author David Masclet
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class GisgraphySearchEntry {
+
+
+	private static List<String> NAME_HOUSE_COUNTRYCODE = new ArrayList<String>() {
```

Comment:
Is this still used, if not, can we remove it?

Reply:
yes. Removed.